From d81de02c5044e99bca26dba73654457d6084618d Mon Sep 17 00:00:00 2001
From: Oleg Kolosov <bazurbat@gmail.com>
Date: Fri, 30 Mar 2017 21:16:24 +0200
Subject: [PATCH] Add USE_DLT option to CMakeLists

This patch allows to disable the usage of DLT even if it is found during
configure.
---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6c78c9d..331e809 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,6 +68,9 @@ message(STATUS "USE_FILE is set to value: ${USE_FILE}")
 OPTION(USE_CONSOLE "Set to OFF to disable console logging" OFF )
 message(STATUS "USE_CONSOLE is set to value: ${USE_CONSOLE}")
 
+OPTION(USE_DLT "Enable DLT" OFF)
+message(STATUS "USE_DLT is set to value: ${USE_DLT}")
+
 # Make relative paths absolute (needed later on)
 foreach(p LIB INCLUDE CMAKE)
   set(var INSTALL_${p}_DIR)
@@ -103,11 +106,13 @@ ENDIF(MSVC)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DCOMMONAPI_LOGLEVEL=COMMONAPI_LOGLEVEL_${MAX_LOG_LEVEL}")
  
+IF(USE_DLT)
 FIND_PACKAGE(PkgConfig)
 pkg_check_modules(DLT "automotive-dlt >= 2.11")
 IF(DLT_FOUND) 
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_DLT")
 ENDIF(DLT_FOUND)
+ENDIF(USE_DLT)
 
 ##############################################################################
 
-- 
2.10.2

